src/Tizen.Wearable.CircularUI.Forms.Renderer/Shell/NavigationDrawer.cs

```diff
@@ -0,0 +1,305 @@
+
+            _outterBox.Show();
+            var opacity = _outterBox.Opacity;
+
+            new Animation((progress) =>
+            {
+                _outterBox.Opacity = opacity + (int)((255 - opacity) * progress);
+
+            }).Commit(_animatable, "FadeIn", length: length, finished: (p, e) =>
+            {
+                _outterBox.Opacity = 255;
+                tcs.SetResult(true);
+            });
+
+            return await tcs.Task;
+        }
+
+        public async Task<bool> HideAsync(EvasObject target, Easing easing = null, uint length = 300)
+        {
+            var tcs = new TaskCompletionSource<bool>();
+
+            var opacity = _outterBox.Opacity;
+            new Animation((progress) =>
+            {
+                _outterBox.Opacity = opacity - (int)(progress * opacity);
+
+            }).Commit(_animatable, "FadeOut", length: length, finished: (p, e) =>
+            {
+                _outterBox.Opacity = 0;
+                _outterBox.Hide();
+                tcs.SetResult(true);
+            });
+
+            return await tcs.Task;
+        }
+
+        protected override IntPtr CreateHandle(EvasObject parent)
+        {
+            _gestureBox = new Box(parent);
+            return _gestureBox.Handle;
+        }
+
+        void Initialize()
+        {
+            AlignmentX = -1;
+            AlignmentY = -1;
+            WeightX = 1;
+            WeightY = 1;
+
+            _gestureBox.RepeatEvents = true;
+
+            _outterBox = new Box(this);
+            _outterBox.Show();
+
+            _contentBox = new Box(this);
+            _contentBox.PropagateEvents = false;
+
+            _label = new ElmSharp.Label(_outterBox)
+            {
+                BackgroundColor = EColor.Pink,
+            };
+
+            _contentBox.Show();
+            _label.Show();
+
+            _outterBox.PackEnd(_contentBox);
+            _outterBox.PackEnd(_label);
+
+            _label.StackAbove(_contentBox);
+
+            _gestureLayer = new GestureLayer(_gestureBox);
+            _gestureLayer.Attach(_gestureBox);
+
+            _gestureLayer.SetMomentumCallback(GestureLayer.GestureState.Start, OnStart);
+            _gestureLayer.SetMomentumCallback(GestureLayer.GestureState.End, OnEnd);
+            _gestureLayer.SetMomentumCallback(GestureLayer.GestureState.Abort, OnEnd);
+
+            _dragGestureLayer = new GestureLayer(_outterBox);
+            _dragGestureLayer.Attach(_outterBox);
+
+            _dragGestureLayer.SetMomentumCallback(GestureLayer.GestureState.Move, OnDragged);
+            _dragGestureLayer.SetMomentumCallback(GestureLayer.GestureState.End, OnDragEnd);
+            _dragGestureLayer.SetMomentumCallback(GestureLayer.GestureState.Abort, OnDragEnd);
+
+            _gestureBox.SetLayoutCallback(OnLayout);
+            _outterBox.SetLayoutCallback(OnContentLayout);
+
+            RotaryEventManager.Rotated += OnRotate;
+        }
+
+        async void OnRotate(EventArgs args)
```

Comment:
How about change name to `OnRotaryEventReceived`

Reply:
Fixed.